newIDE/app/src/AssetStore/index.js

```diff
@@ -300,6 +302,16 @@ export const AssetStore = ({ project }: Props) => {
                     }
                   />
                 )}
+                {isOnHomePage && error && (
+                  <PlaceholderError>
```

Comment:
should we pass an `onRetry` to the component with the function `fetchAssetsAndFilters`

Reply:
Good idea! I'll add it